ui/app/pages/swaps/select-quote-popover/index.scss

```diff
@@ -143,7 +132,37 @@
   &__receiving {
```

Comment:
I rearranged all of the column classes to be adjacent, to make it easier to compare them